plugins/faustwp/includes/replacement/callbacks.php

```diff
@@ -27,26 +29,42 @@
  * @param string $content The post content.
  *
  * @return string The post content.
- * @todo Needs work...
  */
 function content_replacement( $content ) {
```

Comment:
This function now attempts to rewrites links in post_content once. Previously they were rewritten up to 3 times: 1) in `content_replacement` 2) in `image_source_replacement` 3) and in `url_replacement` for the GraphQL responses.